src/cljs/rems/navbar.cljs

```diff
@@ -100,7 +108,8 @@
   (let [identity @(rf/subscribe [:identity])]
     [:div.fixed-top
      [skip-navigation]
-     [:div.navbar-top-bar [:div.navbar-top-left] [:div.navbar-top-right]]
+     [:div.navbar-top-bar
+      [:div.navbar-top-left] [:div.navbar-top-right]]
```

Comment:
I think we should either keep the old format or have `top-left` and `top-right be on their own lines. This is kind of in the middle of those two.

Reply:
hmmm, ok, I am pretty sure cljfmt fixed it, but I can try